Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,202 @@
 /*
-Project Name: HTML Template
-Client: Your Client
-Author: Your Name
-Developer: @poopsplat in Chicago
-*/
+Project Name: Relaxr Landing Page
+Client: Relaxr
+Author: Sara V.
+Developer: SKV */
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+
+.clearfix {
+  *zoom: 1;
+}
+
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
 
 /******************************************
 /* BASE STYLES
 /*******************************************/
+
+/*Note: I adjusted some of the font weights and sizes from the design_file 
+directions because they didn't look right on my mockup. Was this happening
+for others? I assume different browsers/OSs render fonts and sizes differently?*/
+
 body {
+  width: 50%;
   color: black;
+  opacity: .85;
+  margin: 0 auto;
   font-size: 16px;
-  background: white;
+  font-family: 'open sans', sans-serif;
 }
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+header {
+	width: 100%;
+	height: 425px;
+	background-size: cover; /*The word 'cover' didn't change color, which made me think the code was incorrect/broken. Do you know why this happened?*/
```

Comment:
There a handful of properties and values that are _newer_, though widely supported and safe to use, that for whatever reason are not totally syntax highlighted, but your cover value is working just fine, despite it not being highlighted properly in your text editor, so no worries! :)